json_merger/dict_merger.py

```diff
@@ -139,14 +139,14 @@ def conflicts(self):
     def _merge_base_values(self):
         if self.head == self.update:
             self.merged_root = self.head
-        elif self.head == NOTHING:
-            self.merged_root = self.update
         elif self.update == NOTHING:
             self.merged_root = self.head
```

Comment:
isn't this the same issue?

Reply:
Yes, but I didn't know if I should change it too

Reply:
yes, I think you should.